pennylane/gradients/parameter_shift_cv.py

```diff
@@ -814,3 +816,26 @@ def processing_fn(results):
         return jacobian
 
     return gradient_tapes, processing_fn
+
+
+def _get_output_dim(tape):
+    """Update the dimension of the output of the quantum script.
+
+    Sets:
+        self._output_dim (int): Size of the quantum script output (when flattened)
+
+    This method makes use of `self.batch_size`, so that `self._batch_size`
+    needs to be up to date when calling it.
+    Call `_update_batch_size` before `_update_output_dim`
+    """
+    _output_dim = 0
+    for m in tape.measurements:
+        # attempt to infer the output dimension
+        if isinstance(m, ProbabilityMP):
+            _output_dim += m.shape()
+        elif not isinstance(m, StateMP):
+            _output_dim += 1
+    if tape.batch_size:
+        _output_dim *= tape.batch_size
+
+    return _output_dim

```

Comment:
Actually, I just double-checked, and `default.gaussian` doesn't even support more than one observable 🤦 . We might just be able to replace any use of `output_dim` with `1`.

Reply:
True true. Codecov also saide L835 L839 not covered

Reply:
I see this was marked as resolved, but it's not clear to me what the outcome was. Is there any circumstance where `output_dim` isn't just 1 for this transform? Should we add a validation check that there's only a single measurement and just use `1` instead of adding a function for this?

Reply:
https://github.com/PennyLaneAI/pennylane/blob/eefe6efa352ebccf4ea30127779670f52e15ebb5/pennylane/gradients/parameter_shift_cv.py#L698

Reply:
I'd be happy with just replacing any use of `output_dim` with `1`.